src/main/java/Duke.java

```diff
@@ -1,10 +1,229 @@
+//        AnchorPane.setRightAnchor(sendButton, 1.0);
+//
+//        AnchorPane.setLeftAnchor(userInput , 1.0);
+//        AnchorPane.setBottomAnchor(userInput, 1.0);
+//
+//        welcome();
+//        //Step 3. Add functionality to handle user input.
+//        sendButton.setOnMouseClicked((event) -> {
+//            dialogContainer.getChildren().add(getDialogLabel(userInput.getText()));
+//            userInput.clear();
+//        });
+//
+//        userInput.setOnAction((event) -> {
+//            dialogContainer.getChildren().add(getDialogLabel(userInput.getText()));
+//            userInput.clear();
+//        });
+//
+//        //Scroll down to the end every time dialogContainer's height changes.
+//        dialogContainer.heightProperty().addListener((observable) -> scrollPane.setVvalue(1.0));
+//
+//        //Part 3. Add functionality to handle user input.
+//        sendButton.setOnMouseClicked((event) -> {
+//            try {
+//                handleUserInput();
+//            } catch (DukeException e) {
+//                e.printStackTrace();
+//            }
+//        });
+//
+//        userInput.setOnAction((event) -> {
+//            try {
+//                handleUserInput();
+//            } catch (DukeException e) {
+//                e.printStackTrace();
+//            }
+//        });
+//    }
+
+//    /**
+//     * Iteration 1:
+//     * Creates a label with the specified text and adds it to the dialog container.
+//     * @param text String containing text to add
+//     * @return a label with the specified text that has word wrap enabled.
+//     */
+//    private Label getDialogLabel(String text) {
+//        // You will need to import `javafx.scene.control.Label`.
+//        Label textToAdd = new Label(text);
+//        textToAdd.setWrapText(true);
+//
+//        return textToAdd;
+//    }
+
+//    /**
+//     * Iteration 2:
+//     * Creates two dialog boxes, one echoing user input and the other containing Duke's reply and then appends them to
+//     * the dialog container. Clears the user input after processing.
+//     */
+//    private void handleUserInput() throws DukeException {
+//        run(userInput.getText());
+//        String userText = this.input;
+//        String dukeText = getResponse(this.output);
+//        dialogContainer.getChildren().addAll(
+//                DialogBox.getUserDialog(userText, user),
+//                DialogBox.getDukeDialog(dukeText, duke)
+//        );
+//        userInput.clear();
+//    }
+//
+    private void welcome() throws IOException {
+        output = "Hello! I'm Duke\n" + "What can I do for you?";
+        String dukeText = "Welcome!\n" + this.output;
+        dialogContainer.getChildren().add(
+                DialogBox.getDukeDialog(dukeText, duke)
+        );
+        userInput.clear();
+    }
+
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
```

Comment:
Should the method header comment start with a verb instead? e.g. "Gets the response"